source/includes/_assets.md

```diff
@@ -232,6 +229,8 @@ Creates a new asset for the specified entity in the URL. Returns the created
 asset, along with any validation errors and warnings. The required
 [scope](#api-authorization-oauth-scopes) is `write:assets`.
 
+The `year` in _annual_data_ is required along with `asset_size`,`property_type_code` and `asset_name`. You can pass data for the past 4 years. If no record for that year is available, a new one will be created. Old records will be updated but won't have any effect on past surveys and rankings.
```

Comment:
I would provide an example for the `past 4 years`, e.g.:

> If you're reporting in 2020, you can provide data for years 2016-2019

Also, add a couple of line breaks, don't have 200-char long lines ;)